tests/test_bilstm.py

```diff
@@ -21,6 +23,28 @@ def test_vanilla():
     assert model.score(X, Y) > 0.6
 
 
+def test_save_load():
+    X = [
+        "One",
+        "One only",
+        "Two nothing else",
+        "Two and three"
+    ]
+    Y = np.array([0, 0, 1, 1])
+
+    vec = KerasVectorizer()
+    X_vec = vec.fit_transform(X)
+
+    model = BiLSTMClassifier()
+    model.fit(X_vec, Y)
+
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        model.save(tmp_dir)
+        loaded_model = BiLSTMClassifier()
+        loaded_model.load(tmp_dir)
+        assert loaded_model.score(X_vec, Y) > 0.6
```

Comment:
if this doesn't pass it might just be because there is a problem with another part of the BiLSTMClassifier not necessarily the save/load bit, could you just do `assert loaded_model`?

Reply:
you could still also `assert loaded_model.score(X_vec, Y) > 0.6` to make sure it's also loaded correctly

Reply:
hmm, i see the point. i guess what i am trying to show is that there is a trained model loaded unlike an empty one. if i assert loaded_model this will pass regardless. maybe i can do assert loaded_model.model which would indicate that there is a model which is not present purely when you initialise `BiLSTMClassifier`

Reply:
76821eada91fcc2ed1930698298621e1dd6d06f7